transport/transporters/rabbitmq/config.go

```diff
@@ -0,0 +1,52 @@
+/*
+  Copyright 2019 Nextdoor.com, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package rabbitmq
+
+import (
+	"gopkg.in/Nextdoor/cli.v1"
+	"gopkg.in/Nextdoor/cli.v1/altsrc"
+)
+
+const (
+	ConfVarExchangeName   = "exchange-name"
+	ConfVarURL            = "rabbitmq-url"
+	ConfVarWriteBatchSize = "rabbitmq-batch-size"
+)
+
+var Flags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "config",
+		Value: "config.yaml",
+		Usage: "bifrost YAML config file",
+	},
+	altsrc.NewStringFlag(cli.StringFlag{
+		Name:   ConfVarExchangeName,
+		Usage:  "RabbitMQ Exchange Name",
+		EnvVar: "BIFROST_RABBITMQ_EXCHANGE_NAME",
+	}),
+	altsrc.NewStringFlag(cli.StringFlag{
+		Name:   ConfVarURL,
+		Usage:  "RabbitMQ URL",
+		EnvVar: "BIFROST_RABBITMQ_URL",
```

Comment:
@diranged what are your thoughts about prefixed transport config vars? Looks like `RABBITMQ_URL` is a standard URL but `RABBITMQ_EXCHANGE_NAME` isn't. And certainly neither is `RABBITMQ_BATCH_SIZE`. Question is do we prefix everything, nothing, or anything custom (non-standard). Note we do similar in the kinesis transport with aws vars and stream name.

Reply:
Re `RABBITMQ_BATCH_SIZE` - couldn't we do someting more generic that spans different transporters, like `BATCH_SIZE`?

Re the Exchange name.. is that the same as the Virtual Host thats part of the URI specification: https://www.rabbitmq.com/uri-spec.html ?

Reply:
The exchange name is completely different from the virtual host and it wouldn't be appropriate to include it in the URI.  I could, but I've never seen any other RabbitMQ client do so.

Reply:
@taybin,
  Fair enough - I was just checking. I am fine with us adding variables that don't have defined standards. Just want to try to use standards where we can.